.circleci/config.yml

```diff
@@ -106,44 +106,24 @@ jobs:
       - run:
           name: deploy to cloud.gov
           command: |
-            # If we do not specify a space, then deploy to the branch that we are on.
```

Comment:
Removing this code comment because it doesn't strike me as correct ... the branches that trigger deploys are limited by the `only` filter at the bottom of the file, it's not possible to deploy from any branch.